lib/compat/wordpress-6.5/class-wp-script-modules.php

```diff
@@ -226,6 +228,27 @@ public function print_import_map() {
 			}
 		}
 
+		/**
+		 * Prints the necessary script to load import map polyfill for browsers that
+		 * do not support import maps.
+		 *
+		 * TODO: Replace the polyfill with a simpler version that only provides
+		 * support for import maps and load it only when the browser doesn't support
+		 * import maps (https://github.com/guybedford/es-module-shims/issues/406).
+		 *
+		 * @since 6.5.0
+		 */
+		public function print_import_map_polyfill() {
+			$import_map = $this->get_import_map();
+			if ( ! empty( $import_map['imports'] ) ) {
+				wp_print_script_tag(
+					array(
+						'src'   => defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN ? gutenberg_url( '/build/modules/importmap-polyfill.min.js' ) : includes_url( 'js/dist/importmap-polyfill.min.js' ),
```

Comment:
@luisherranz I've put a folder for WP Core where there are several `wp-polyfill-...`files. 
Feel free to change it if it is not ok.

Reply:
Is there a folder like that in Gutenberg?

Reply:
Yep. There are a few polyfills.

Reply:
I mean, is `/build/modules/` the right folder for this polyfill in Gutenberg?

Reply:
It's the folder in the webpack config done by Interactivity.

https://github.com/WordPress/gutenberg/blob/1e94019ed6e8d5a785625451a9179e81dc5f8303/tools/webpack/interactivity.js#L80

There is a folder `build/vendors` which has `inert-polyfill.js` and `inert-polyfill.min.js`. Maybe we can move it there.

I didn't found any other polyfill outside rather than one for testing and another one for React Native.